utils/progressbar/progressbarmng.go

```diff
@@ -289,6 +301,9 @@ func filterColor(color Color) (filter string) {
 	default:
 		filter = "⬜"
 	}
+	if windows {
+		filter = "●"
+	}
 	return
```

Comment:
The code in this function can be made a bit more efficient when running on Windows by changing it to - 
```
	if windows {
		filter = "●"
                return
	}
	switch color {
	case GREEN:
		filter = "🟩"
	case WHITE:
		filter = "⬜"
	default:
		filter = "⬜"
	}
	return
```